src/yall.mjs

```diff
@@ -3,27 +3,31 @@ export default function (options) {
     options = {};
   }
 
-  const intersectionObserverSupport = "IntersectionObserver" in window && "IntersectionObserverEntry" in window;
+  const doc = document;
```

Comment:
I like these. They're still readable, but the uglifier will do the same thing with them as my abbreviations did.